### DISCUSSION TITLE
Effects of expanding paraDomain in ERT inversion

### DISCUSSION THREAD

**Discussion by benbpete** [gimli-org/pyGIMLi#471] [Q&A]
Hi All,

I've been playing around with ERT modeling and came across an effect I don't understand.  Below are two models (full paraDomain extents, and then a smaller scale figure) of the same dipole-dipole data set, the only difference is when creating the PLC/Mesh. For the small scale figures, I adjusted the color scales to be the same. Finally attached is the apparent resistivity psuedosection. 

For the first model I used the default paraBoundary =2, and for the second I used paraBoundary = 24.  The result in the final model is pretty large.  Both models converge very quickly with chi^2 of 0.18 and 0.1.

The inversion is simply:
`mod = ert.invert(data, mesh=mesh, zWeight=1, lam = 100, verbose=True, robustData = False)`

I can attach my code/data if it makes any difference. Any thoughts?

![DD__psuedosection](https://user-images.githubusercontent.com/105517351/207325837-8a1236fd-bb05-4c30-9623-fda2f041a359.png)

![DD__model_PB2](https://user-images.githubusercontent.com/105517351/207325190-4f08f484-3636-4504-8ef7-8eddcc691f26.png)

![DD__model_PB2_small](https://user-images.githubusercontent.com/105517351/207326202-8425378c-066b-4ce9-b3d7-b9f2144a8425.png)

![DD__model_PB24](https://user-images.githubusercontent.com/105517351/207325233-1da802fe-202e-4d40-ac8f-2d1c915673e1.png)

![DD__model_PB24_small](https://user-images.githubusercontent.com/105517351/207325245-cc984ec4-ffdf-4dfe-a2b7-b57c279e3f52.png)

Thanks!
Ben

**Comment by halbmy** [comment]
All the models are equivalently fitting the data and show a blurred two-layer model. Outside of the area of resolution the model is arbitrary and depends on resolution.

However, with a 100m surface layout you will not be able to resolve anything beyond 30m depth, so the models are all too deep (and some too wide). Seems like the alpha-shading heuristics makes only sense for models of reasonable size.

**Reply by benbpete** [reply]
Yeah, what I am doing is just exploring how changes in the paraDomain size affect the final model.  It was good to see how using the sensitivity as a guide for model limits really changes depending on the size of the paraDomain and that it isn't reliable with overly large modeling domains.

Just so I am thinking about this correctly, even though the models are equivalently fitting, the reason for the differences is likely a combination of 1) larger paraDomain which results a larger distance to the model boundaries and 2) change in triangular mesh size near the left/right boundaries of the paraDomain and 3) likely a large number of equivalent solutions because the resistivity contrast in the "true model" is relatively small and gradational. The sensitivity is also spread out over a large area, which I am guessing results in a larger "domain" of equivalent solutions. Anything else that might be affecting it?
